src/subscription_manager/gui/widgets.py

```diff
@@ -449,14 +527,16 @@ class ContractSubDetailsWidget(SubDetailsWidget):
                      "virt_only_text",
                      "details_view"]
 
-    glade_file = "subdetailscontract.glade"
+    gui_file = "subdetailscontract"
 
     def __init__(self, product_dir):
         super(ContractSubDetailsWidget, self).__init__(product_dir)
         # Save the original background color for the
         # start_end_date_text widget so we can restore it in the
         # clear() function.
-        self.original_bg = self.start_end_date_text.rc_get_style().base[gtk.STATE_NORMAL]
+        # FIXME
+        #self.original_bg = self.start_end_date_text.rc_get_style().base[ga_Gtk.StateType.NORMAL]
+        # FIXME
```

Comment:
That's still kind of broken. There are two things related.

1) rc_get_style() doesn't exist, and all the widget style stuff is moved to GtkStyleContext, that is more CSS like. So changing stuff like the old code wants a GtkStyleContextProvider, and the 'reading' the applied style needs to use GtkSettings.

So we need a compatibility/abstraction layer there that hasn't been written yet.

2) I can't point to the a particular doc atm, but Gtk3 guidelines suggest not trying to have the application set or change widget styles or colors, and if required doing it sort of like CSS, and add properties to the widget and add/create a StyleProvider that themes that particular style. 

My vote would be to just remove the code that attempts to tweak the colors since it's unlikely anyone will notice.